core/templates/dev/head/pages/exploration_player/StatsReportingService.js

```diff
@@ -46,17 +46,17 @@ oppia.constant('STATS_REPORTING_URLS', {
 });
 
 oppia.factory('StatsReportingService', [
-  '$http', '$interval', 'StopwatchObjectFactory', 'MessengerService',
-  'UrlInterpolationService', 'STATS_REPORTING_URLS', 'siteAnalyticsService',
-  'STATS_EVENT_TYPES', 'ExplorationContextService', 'PAGE_CONTEXT',
-  'DEFAULT_OUTCOME_CLASSIFICATION', 'PlaythroughService',
-  'ENABLE_PLAYTHROUGH_RECORDING',
+  '$http', '$interval', 'ExplorationContextService', 'MessengerService',
+  'PlaythroughService', 'siteAnalyticsService', 'UrlInterpolationService',
```

Comment:
`StopwatchObjectFactory` and `UrlInterpolationService` need to be swapped here.

Reply:
@brianrodri Actually, they need to be swapped below. Don't Factories come after Services?